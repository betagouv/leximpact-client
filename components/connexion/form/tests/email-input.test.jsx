import { createShallow } from "@material-ui/core/test-utils";

import EmailInput from "../email-input";

describe("components | connexion | email-input", () => {
  let shallow;
  beforeAll(() => {
    shallow = createShallow();
  });
  describe("snapshot", () => {
    it("doit correspondre avec les props requises", () => {
      const props = {
        domains: ["@any-domain"],
      };
      const wrapper = shallow(<EmailInput {...props} />);
      expect(wrapper).toBeDefined();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
