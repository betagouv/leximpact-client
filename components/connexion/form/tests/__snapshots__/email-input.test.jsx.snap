// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | connexion | email-input snapshot doit correspondre avec les props requises 1`] = `
<EmailTextInput
  classes={
    Object {
      "fieldsContainer": "EmailTextInput-fieldsContainer-1",
      "formGroup": "EmailTextInput-formGroup-2",
      "formLabel": "EmailTextInput-formLabel-3",
      "inputContainer": "EmailTextInput-inputContainer-4",
      "inputField": "EmailTextInput-inputField-5",
      "inputRoot": "EmailTextInput-inputRoot-6",
      "selectRoot": "EmailTextInput-selectRoot-7",
    }
  }
  domains={
    Array [
      "@any-domain",
    ]
  }
/>
`;
